src/sass/mixins/_Directionality.Mixins.scss

```diff
@@ -32,292 +32,292 @@
+@mixin ms-borderColor($top, $right, $bottom, $left) {
   border-color: $top $right $bottom $left;
 
-  @include RTL {
+  @include ms-RTL {
     border-color: $top $left $bottom $right;
   }
 }
 
-@mixin border-left($width, $style, $color) {
-  @include LTR {
+@mixin ms-borderLeft($width, $style, $color) {
+  @include ms-LTR {
     border-left: $width $style $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-right: $width $style $color;
   }
 }
 
-@mixin border-left-color($color) {
-  @include LTR {
+@mixin ms-borderLeftColor($color) {
+  @include ms-LTR {
     border-left-color: $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-right-color: $color;
   }
 }
 
-@mixin border-left-style($style) {
-  @include LTR {
+@mixin ms-borderLeftStyle($style) {
+  @include ms-LTR {
     border-left-style: $style;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-right-style: $style;
   }
 }
 
-@mixin border-left-width($width) {
-  @include LTR {
+@mixin ms-borderLeftWidth($width) {
+  @include ms-LTR {
     border-left-width: $width;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-right-width: $width;
   }
 }
 
-@mixin border-radius($topLeft, $topRight, $bottomRight, $bottomLeft) {
+@mixin ms-borderRadius($topLeft, $topRight, $bottomRight, $bottomLeft) {
   border-radius: $topLeft $topRight $bottomRight $bottomLeft;
 
-  @include RTL {
+  @include ms-RTL {
     border-radius: $topRight $topLeft $bottomLeft $bottomRight;
   }
 }
 
-@mixin border-right($width, $style, $color) {
-  @include LTR {
+@mixin ms-borderRight($width, $style, $color) {
+  @include ms-LTR {
     border-right: $width $style $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-left: $width $style $color;
   }
 }
 
-@mixin border-right-color($color) {
-  @include LTR {
+@mixin ms-borderRightColor($color) {
+  @include ms-LTR {
       border-right-color: $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
       border-left-color: $color;
   }
 }
 
-@mixin border-right-style($style) {
-  @include LTR {
+@mixin ms-borderRightStyle($style) {
+  @include ms-LTR {
     border-right-style: $style;
   }
   
-  @include RTL {
+  @include ms-RTL {
     border-left-style: $style;
   }
 }
 
-@mixin border-right-width($width) {
-  @include LTR {
+@mixin ms-borderRightWidth($width) {
+  @include ms-LTR {
     border-right-width: $width;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-left-width: $width;
   }
 }
 
 // Floats.
-@mixin clear($side) {
+@mixin ms-clear($side) {
   @if $side == left {
-    @include LTR {
+    @include ms-LTR {
       clear: $side;
     }
 
-    @include RTL {
+    @include ms-RTL {
       clear: right;
     }
   } @else if $side == right {
-    @include LTR {
+    @include ms-LTR {
       clear: $side;
     }
 
-    @include RTL {
+    @include ms-RTL {
       clear: left;
     }
   } @else {
       clear: $side;
   }
 }
 
-@mixin float($direction) {
+@mixin ms-float($direction) {
```

Comment:
It's never fun when we introduce breaking changes, but it is a simple find-replace in this case. There's no change in functionality so once all of the build errors are caught it's good to go. I'm planning on doing the update in Fabric React and can help if other teams have concerns.